modules/react/select/lib/SelectInput.tsx

```diff
@@ -61,28 +73,23 @@ export const SelectInput = createSubcomponent(TextInput)({
     model
   ) => {
     return (
-      <InputGroup data-width="ck-formfield-width">
+      <InputGroup data-width="ck-formfield-width" {...selectInputStencil()}>
         {inputStartIcon && model.state.selectedIds.length > 0 && (
-          <InputGroup.InnerStart data-part="select-start-icon-container" {...selectIconsStencil()}>
-            <SystemIcon data-part="select-start-icon" icon={inputStartIcon} />
+          <InputGroup.InnerStart {...selectInputStencil.parts.startIconContainer}>
+            <SystemIcon {...selectInputStencil.parts.startIcon} icon={inputStartIcon} />
           </InputGroup.InnerStart>
         )}
         {/* Hidden input to handle ids */}
-        <InputGroup.Input
-          data-part="select-hidden-input"
-          {...formInputProps}
-          {...hiddenSelectInputStencil()}
-        />
+        <InputGroup.Input {...selectInputStencil.parts.hiddenInput} {...formInputProps} />
         {/* Visual input */}
         <InputGroup.Input
           as={Element}
           placeholder={placeholder}
-          data-part="select-visual-input"
-          {...elemProps}
-          {...mergeStyles(elemProps, selectInputStencil())}
+          {...selectInputStencil.parts.visualInput}
+          {...handleCsProp(elemProps)}
```

Comment:
wouldn't this be:
`{...handleCsProp(elemProps, selectInputStencil.parts.visualInput)}`

Reply:
oooooooh, now i see

Reply:
it feels odd, IMO